scripts/core/interactive-article-actions-panel/utils/get-initial-destination.ts

```diff
@@ -85,12 +85,12 @@ export function getInitialDestination(
     return destination;
 }
 
-export function getCurrentDeskDestination(): ISendToDestination {
-    let destinationDesk: string = sdApi.desks.getCurrentDeskId();
+export function getCurrentDeskDestination(id?: string): ISendToDestination {
+    const destinationDeskId: string = sdApi.desks.getCurrentDeskId();
```

Comment:
Can you remove this variable and inline this code?

Reply:
okay